### PR TITLE
Adding IT jokes

### PR DESCRIPTION
**Pull request by xani-dev**
Hi, adding a few IT jokes to your DB :)

### CODE REVIEW

**Comment by eoaranda** [comment]
Did you also added extra spacing ?

File: db/data.json [hunk 1]
```diff
@@ -1,62 +1,65 @@
...
+    "Why didn't the vampire attack Taylor Swift? <br><br>She had bad blood.",
+    "What do you get when you cross a snowman with a vampire? <br><br>Frostbite.",
+    "What do you call a lonely cheese? <br><br>Provolone.",
+    "Why can't you hear a pterodactyl go to the bathroom? <br><br> Because the pee is silent.",
+    "What does an angry pepper do? <br><br>It gets jalapeño your face.",
+    "What did the buffalo say to his son when he dropped him off at school? <br><br>Bison.",
+    "What do you call someone with no body and no nose? <br><br>Nobody knows.",
+    "Why did the crab never share? <br><br> Because he's shellfish.",
+    "Why do Java developers wear glasses? Because they can’t C#",
+    "What do you call 8 hobbits? A hobbyte",
```
Comment (eoaranda) [inline db/data.json]: haha this one is pretty good 👍🏼

**Comment by xani-dev** [comment]
> Did you also added extra spacing ?

Auto-formatting did! Just a few jokes added at the end of the list.

File: db/data.json [hunk 2]
```diff
@@ -1,62 +1,65 @@
...
+    "What do prisoners use to call each other? <br><br>Cell phones.",
+    "Why couldn't the bike standup by itself? <br><br>It was two tired.",
+    "What do you call a dog that can do magic? <br><br>A Labracadabrador.",
+    "Why didn't the vampire attack Taylor Swift? <br><br>She had bad blood.",
+    "What do you get when you cross a snowman with a vampire? <br><br>Frostbite.",
+    "What do you call a lonely cheese? <br><br>Provolone.",
+    "Why can't you hear a pterodactyl go to the bathroom? <br><br> Because the pee is silent.",
+    "What does an angry pepper do? <br><br>It gets jalapeño your face.",
+    "What did the buffalo say to his son when he dropped him off at school? <br><br>Bison.",
+    "What do you call someone with no body and no nose? <br><br>Nobody knows.",
```
Comment (eoaranda) [inline db/data.json:59]: All right all looks, good can you please add the format of <br><br> after the joke punchline ? and I will accept them, thanks !

**Comment by eoaranda** [comment]
@xani-dev great, added to the master branch, thanks for the collab.

**Pull request merged**